.drone.star

```diff
@@ -1046,7 +1048,7 @@ def installCore(version, db):
 		'image': 'owncloudci/core',
 		'pull': 'always',
 		'settings': {
-			'version': version,
+			'git_reference': commitID,
```

Comment:
that does not work?
```suggestion
			'git_reference': $CORE_COMMITID,
```

Reply:
no..because we have to  source the .env file which we can only do on steps part